src/utils/interface.js

```diff
@@ -16,15 +19,21 @@ export function chat(chatMessage) {
 }
 
 let AUTONOMOUS = true;
+let VISUALS = true;
```

Comment:
Hmm I should've said this when we wrote the autonomous code, but I don't like the all caps here. Usually all caps means constants